package.json

```diff
@@ -2,6 +2,7 @@
   "name": "coderscrew-app",
   "version": "1.0.0",
   "scripts": {
+    "predev": "docker-compose up -d",
```

Comment:
I'm not sure if we should have this command.
1. What if someone terminates the `dev` process and runs it again. Wouldn't Docker try to start the second time?
2. It's kind of asymmetric because Docker starts automatically but needs to be terminated manually.
I think it could be better if everyone needs to run Docker explicitly from the command line.